scss/tech.scss

```diff
@@ -19,11 +19,17 @@
     position: absolute;
     left: -20px;
     top: -20px;
+    transition: all 1s ease;
```

Comment:
don't use all, use the specific property. in this case `top 1s ease, left 1s ease`